src/init-ceph.in

```diff
@@ -425,6 +431,52 @@ osd_has_blocked_ops() {
     fi
 }
 
+osd_has_stuck_peering() {
```

Comment:
Should this just replace the osd_has_blocked_ops()? I'm wondering if this is the same issue that we saw in Jewel and it is just producing a different message in Mimic. Looking at the code in Mimic there is still a function that produces "ops are blocked" so in theory the above routine is still relevant. I'm just not sure we've seen it triggered. Any chance you've seen this triggered during your testing. If not then we probably need to keep both the blocked_ops and the stuck_peering detection.

Reply:
Didn't see "ops are blocked" during investigation. We shouldn't replace.

Reply:
OK. Tx.